AppInspector.CLI/CLICmdOptions.cs

```diff
@@ -58,8 +83,14 @@ public class CLIAnalyzeCmdOptions : CLICommandOptions
         [Option('t', "tag-output-only", Required = false, HelpText = "Output only identified tags", Default = false)]
         public bool SimpleTagsOnly { get; set; }
 
+        [Option("file-timeout", Required = false, HelpText = "Maximum amount of time in seconds to allow for processing each file.", Default = 0)]
+        public int FileTimeOut { get; set; } = 0;
```

Comment:
Better for this to be in milliseconds, maybe?

Reply:
Sure.